src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -1453,7 +1529,11 @@ public void Process_StartTest()
             Assert.Equal(userName, p.StartInfo.UserName);
             Assert.Same(password, p.StartInfo.Password);
             Assert.Equal(domain, p.StartInfo.Domain);
+
+            // Make sure that process is fully created before killing it to avoid AccessDeniedExceptions.
+            Thread.Sleep(500);
```

Comment:
> This shouldn't repro anymore, this was fixed, please revert this change. All the process tests passed in the latest outerloop run.

The test TestInvalidPriorityClass failed locally on my machine with current source from master. Should I really revert it?